roles/sap_ha_pacemaker_cluster/vars/platform_cloud_aws_ec2_vs.yml

```diff
@@ -21,7 +19,7 @@ __sap_ha_pacemaker_cluster_aws_instances: []
 # The aws cli is currently configured anyway for the standard AWS VIP resource.
 sap_ha_pacemaker_cluster_stonith_default:
   id: "res_fence_aws"
-  agent: "stonith:fence_aws"
+  agent: "{{ sap_ha_pacemaker_cluster_stonith_agent_dict.cloud_aws }}"
```

Comment:
I would keep the dict definition including cloud platform details per OS in the OS vars files. 
As soon as there is a more granular differentiation based on OS major or even minor releases required, it will become harder to manage in the platform vars.

For a user actually reading the platform vars, finding the dict definition of a var lookup in the role should not be too hard either, I hope. :)

Reply:
Will defer also, but we'll need a plan of action long-term to handle the "matrix" of options.

- OS Major/Minor
- Platform
  - STONITH Method
    - Fence Agent/s `(and options)`
  - VIP Method
    - Resource Agent/s `(and options)`